src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java

```diff
@@ -0,0 +1,29 @@
+package de.internetsicherheit.brl.bloxberg.cache.ethereum;
+
+import org.web3j.protocol.core.methods.response.Transaction;
+
+import java.util.ArrayList;
+
+public class ReducedTransObjectArray {
+
+    /**
+     * not being used atm. ignore for now
```

Comment:
They are used in the method `addTransaction()`.

Reply:
which is never called.

Reply:
In this case, don't even add this class to this PR, if it is unused code.

Reply:
deleted class